src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityFeed-test.js

```diff
@@ -219,10 +219,32 @@ describe('elements/content-sidebar/ActivityFeed/activity-feed/ActivityFeed', ()
             },
             { isInputOpen: false },
         );
-
         expect(scrollIntoView).toHaveBeenCalledWith(li);
     });
 
+    test('should not scroll to active feed item when activeFeedItemRef is null', () => {
+        const wrapper = shallow(
```

Comment:
It looks like this is duplicated in a few tests. Can we create a helper method, like `getWrapper`? Same for the `ActiveState` test file.